lib/MusicBrainz/Server/WebService/Serializer/JSON/2/Relationship.pm

```diff
@@ -2,7 +2,6 @@ package MusicBrainz::Server::WebService::Serializer::JSON::2::Relationship;
 use Moose;
 use Hash::Merge qw( merge );
 use List::AllUtils qw( any );
-use String::CamelCase qw( camelize );
```

Comment:
Isn’t used in this file, I found it because I grepped for `camelize`. Otherwise unrelated.